pilot/pkg/model/authorization.go

```diff
@@ -135,19 +140,30 @@ func (policy *AuthorizationPolicies) ListServiceRoleBindings(ns string) map[stri
 	return rolesAndBindings.Bindings
 }
 
-// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in the given namespace.
-func (policy *AuthorizationPolicies) ListAuthorizationPolicies(ns string, workloadLabels labels.Collection) []Config {
+// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in root namespace and the config namespace.
+func (policy *AuthorizationPolicies) ListAuthorizationPolicies(configNamespace string,
```

Comment:
[nit] It may be cleaner to keep the function as is (may rename and make private), and change this function to call it twice for root then namespace

```
ret = listAuthorizationPolicies(configNamespace, workloadLabels);
if policy.rootNamespace != "" {
   ret = append(ret, listAuthorizationPolicies(policy.rootNamespace, workloadLabels))
}
```

Reply:
I think it's better to merge this to one function, I don't when we will want to only call one of them. Also this is the same convention followed by networking API (e.g. EnvoyFilter CRD)